lib/presentation/profile/widget/profile_picture.dart

```diff
@@ -33,8 +33,9 @@ class ProfilePicture extends StatelessWidget {
       maxRadius: _maxRadius,
       foregroundImage: imageProvider,
       backgroundImage: const AssetImage(
-        'assets/images/logo.png',
+        'assets/images/default_avatar.png',
```

Comment:
If `assets/images/logo.png` is not used anywhere else, it should be deleted. (It is no longer used as the app icon)